interface.go

```diff
@@ -235,6 +235,17 @@ type LoopInQuote struct {
 	CltvDelta int32
 }
 
+// LoopInSwapInfo contains essential information of a loop-in swap after the
+// swap is initiated.
+type LoopInSwapInfo struct { // nolint
```

Comment:
What lint failure does this trigger?

Reply:
"type name will be used as loop.LoopInSwapInfo by other packages, and that stutters; consider calling this InSwapInfo"

Similar lint error for other types in this file too, but they are silenced by the reference point.